api/v1alpha1/values_types.go

```diff
@@ -0,0 +1,670 @@
+}
+
+// Configuration for each of the supported tracers.
+type TracerConfig struct {
+	// Configuration for the datadog tracing service.
+	Datadog *TracerDatadogConfig `json:"datadog,omitempty"`
+	// Configuration for the lightstep tracing service.
+	Lightstep *TracerLightStepConfig `json:"lightstep,omitempty"`
+	// Configuration for the zipkin tracing service.
+	Zipkin *TracerZipkinConfig `json:"zipkin,omitempty"`
+	// Configuration for the stackdriver tracing service.
+	Stackdriver *TracerStackdriverConfig `json:"stackdriver,omitempty"`
+}
+
+// Configuration for the datadog tracing service.
+type TracerDatadogConfig struct {
+	// Address in host:port format for reporting trace data to the Datadog agent.
+	Address string `json:"address,omitempty"`
+}
+
+// Configuration for the lightstep tracing service.
+type TracerLightStepConfig struct {
+	// Sets the lightstep satellite pool address in host:port format for reporting trace data.
+	Address string `json:"address,omitempty"`
+	// Sets the lightstep access token.
+	AccessToken string `json:"accessToken,omitempty"`
+}
+
+// Configuration for the zipkin tracing service.
+type TracerZipkinConfig struct {
+	// Address of zipkin instance in host:port format for reporting trace data.
+	//
+	// Example: <zipkin-collector-service>.<zipkin-collector-namespace>:941
+	Address string `json:"address,omitempty"`
+}
+
+// Configuration for the stackdriver tracing service.
+type TracerStackdriverConfig struct {
+	// enables trace output to stdout.
+	Debug bool `json:"debug,omitempty"`
+	// The global default max number of attributes per span.
+	MaxNumberOfAttributes uint32 `json:"maxNumberOfAttributes,omitempty"`
+	// The global default max number of annotation events per span.
+	MaxNumberOfAnnotations uint32 `json:"maxNumberOfAnnotations,omitempty"`
+	// The global default max number of message events per span.
+	MaxNumberOfMessageEvents uint32 `json:"maxNumberOfMessageEvents,omitempty"`
+}
+
+// Configuration for base component
+type BaseConfig struct {
+	// For Helm2 use, adds the CRDs to templates.
+	EnableCRDTemplates bool `json:"enableCRDTemplates,omitempty"`
+	// URL to use for validating webhook.
+	ValidationURL string `json:"validationURL,omitempty"`
+	// For istioctl usage to disable istio config crds in base
+	EnableIstioConfigCRDs bool `json:"enableIstioConfigCRDs,omitempty"`
+	ValidateGateway       bool `json:"validateGateway,omitempty"`
+}
+
+type IstiodRemoteConfig struct {
+	// URL to use for sidecar injector webhook.
+	InjectionURL string `json:"injectionURL,omitempty"`
+	// Path to use for the sidecar injector webhook service.
+	InjectionPath string `json:"injectionPath,omitempty"`
+}
+
+type Values struct {
+	Cni    *CNIConfig    `json:"cni,omitempty"`
+	Global *GlobalConfig `json:"global,omitempty"`
+	Pilot  *PilotConfig  `json:"pilot,omitempty"`
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +kubebuilder:validation:Schemaless
+	Ztunnel map[string]string `json:"ztunnel,omitempty"`
```

Comment:
Is there a schema for this? If not, can we create one?

At the very least, this should probably be a `map[string]any`.

Reply:
We decided to remove all the Ambient related fields for now